src/backend/parser/gram.y

```diff
@@ -15551,10 +15551,16 @@ func_expr_common_subexpr:
 				}
 			| TRIM '(' trim_list ')'
 				{
-					$$ = (Node *) makeFuncCall(SystemFuncName("btrim"),
-											   $3,
-											   COERCE_SQL_SYNTAX,
-											   @1);
+					if (sql_dialect == SQL_DIALECT_TSQL)
+						$$ = (Node *) makeFuncCall(list_make2(makeString("sys"), makeString("trim")),
+													$3,
+													COERCE_EXPLICIT_CALL,
```

Comment:
what is meaning of this COERCE_EXPLICIT_CALL and COERCE_SQL_SYNTAX?

Reply:
`COERCE_EXPLICIT_CALL` means display `FuncExpr` as function call, whereas `COERCE_SQL_SYNTAX` means display `FuncExpr` with SQL-mandated special syntax.

Reply:
Why not keep the coercion form same as `COERCE_SQL_SYNTAX`? After-all, it is considered a special syntax in T-SQL as well.

Reply:
removed this check.